week2/homework/src/index.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+let fs = require('fs');
+let args = process.argv.slice(2);
+// console.log(args);
+
+let command = args[0];
+let todoItem = args[1];
+
+if (command === undefined || 'help') {
```

Comment:
This is always evaluated to `true`. You can read it this way

    if ((command === undefined) || 'help') {
      // ...
    }

It should be re-written

    if (command === undefined || command === 'help') {
      // ...
    }

Or ` (!command || command === 'help') `

Reply:
I did not try that one, I did it in the last step, Thanks for the correction.